extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -176,7 +176,8 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
             ignoreHeaders:[
                 "Authorization", 
                 "X-API-Key",
-                "WWW-Authenticate"]
+                "WWW-Authenticate"],
+            addContextOnRequests: undefined
```

Comment:
As this is specific for ajax requests we should probably target the name to that
perhaps
- addAjaxContext
- addAjaxProperties
- etc

We don't want to get too big on the name as this will be public and unminifiable, but we also want it to be specific / recognizable... Trying to avoid the what does that mean questions when some sees it.